components/admin/data/widgets/table/component.js

```diff
@@ -35,20 +36,30 @@ class WidgetsTable extends PureComponent {
     pagination: INITIAL_PAGINATION,
     loading: true,
     widgets: [],
-    filters: { name: null }
+    filters: { name: null, 'úser.role': 'ADMIN' }
```

Comment:
`user.role` not `úser.role` 😛 Are you testing me? 😂

Reply:
you spotted it! 🔍 😅